src/shared/tests.h

```diff
@@ -3,6 +3,19 @@
 
 #include <stdbool.h>
 
+#include "macro.h"
+
+static inline bool manager_errno_skip_test(int r) {
+        return IN_SET(abs(r),
+                      EPERM,
+                      EACCES,
+                      EADDRINUSE,
+                      EHOSTDOWN,
```

Comment:
btw, one of those days it would be great if these non obvious error code would also get a comment why they are listed here